gateway/1.4/modules/ROOT/pages/flex-architecture-uc-overview.adoc

```diff
@@ -0,0 +1,68 @@
+= Integration Use Cases
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+The following integration use cases are determined by the ownership of the API Provider and the ownership of the API Consumer. Each use case also provides physical patterns demonstrating the necessary services in the system. 
+
+For all use case diagrams, Flex Gateway is running in Connected Mode. However, you can also run Flex Gateway in Local Mode for all use cases. You can also use a combination of Connected Mode and Local Mode for separate runtimes in the same or in different environments.
+
+NOTE: The use case diagrams use Kubernetes terminology, but you can extend the architecture to other technology stacks with the appropriate network controls applied.
+
+Flex Gateway supports four use cases:
+
+* xref:flex-architecture-uc1.adoc[]
+* xref:flex-architecture-uc2.adoc[] 
+* xref:flex-architecture-uc3.adoc[] 
+* xref:flex-architecture-uc4.adoc[] 
+
+image:ad-intergration-use-cases.png[The four use cases are linked to products they could support.]
+
+Each use case supports different integration patterns that vary depending on the location of APIs relative to the network Flex Gateway is in. You can use different use case patterns depending on the different API owners and networks in your system. 
+
+In each use case, there are three different classifications of networks. The classifications describe the networks relation to your organization and the network Flex Gateway is in:
+
+* Organization internal: networks owned by your organization and in the same network as Flex Gateway. 
+* Organization external: trusted networks external to your organization but in a different network than Flex Gateway, for example, your organization’s Splunk instance. 
+* Non-Organization external: networks not trusted by your company and not in the same network as Flex Gateway. An API provider’s or API consumers network status affects the necessary security protocols (mTLS) you take when communicating.
+
+If an API Provider or API Consumer is Organization owned, it belongs to either organization internal or organization external networks.
+
+The following table relates the previously mentioned use cases to use case patterns, the networks involved in the pattern, and whether Flex Gateway acts as an ingress or egress contoller or both.  
+
+[%header%autowidth.spread,cols="a,a,a,a,a"]
+|===
+| Use Case | Pattern | Provider Network | Consumer Network | Role of Flex Gateway
```

Comment:
If you want to simplify the table, you could have a separate heading for each use and then have a table that shows only that use case. I think this might be a better way to go.

== UC1:  Organization-Owned API Exposed to an Organization-Owned API Consumer

Pattern    Provider Network    Consumer Network    Role of Flex Gateway

=== UC2: 
Pattern     Provider Network     Consumer Network   Role of Flex Gateway